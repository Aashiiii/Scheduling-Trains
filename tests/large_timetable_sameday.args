Campsie Vineyard "00:03:50"
