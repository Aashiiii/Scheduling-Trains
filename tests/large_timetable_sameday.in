Allawah::Normanhurst::12:23:39
Allawah::Oatley::02:44:55
Ashfield::Padstow::21:10:03
Campsie::Strathfield::03:03:20
Campsie::Sutherland::07:12:19
canterbury::Sydenham::14:45:32
Clyde::Turella::22:10:10
Werrington::Vineyard::11:12:11
Waverton::Vineyard::00:00:01
Campsie::Vineyard::23:45:43
Campsie::Waitara::06:06:06
Gymea:Campsie::12:12:23                                                                                                                                           
Clyde::Windsor::06:06:09
Werrington::Westmead::12:23:43
Waverton::Windsor::08:54:23
Waverton::Gymea::08:54:27
Waverton::Gymea::08:24:23
Waverton::Gymea::08:52:43
Waverton::Heathcote::23:45:22
Waverton::Gymea::03:45:43
Riverwood::Homebush::11:23:32
Oatley::Guildford::01:11:53
Allawah::Normanhurst::20:45:43
Allawah::Oatley::11:03:03
Ashfield::Padstow::03:33:23
Campsie::Strathfield::15:26:03
Campsie::Sutherland::02:32:42
canterbury::Sydenham::02:23:34
Clyde::Turella::03:12:32
Werrington::Vineyard::12:32:34
Waverton::Vineyard::11:42:23
Campsie::Vineyard::09:08:07
Campsie::Waitara::00:07:54
Canterbury::Windsor::02:23:32                                                                                                                                                   
Clyde::Windsor::03:12:25
Werrington::Westmead::21:07:45
Waverton::Windsor::12:12:12
Waverton::Gymea::12:32:12
Waverton::Gymea::09:08:07
Waverton::Gymea::03:12:25
Waverton::Heathcote::03:12:25
Waverton::Gymea::21:07:45
Riverwood::Homebush::09:08:07
Oatley::Guildford::00:07:54
Ashfield::Padstow::03:12:25
Campsie::Strathfield::12:12:12
Campsie::Sutherland::09:08:07
canterbury::Sydenham::12:12:12
Clyde::Turella::06:06:06
Werrington::Vineyard::06:06:06
Waverton::Vineyard::06:06:06
Campsie::Vineyard::00:07:54
Campsie::Waitara::21:07:45
Canterbury::Windsor:: 13:13:24                                                                                                                                                  
Clyde::Windsor::00:07:54
Waverton::Gymea::00::00::00
Waverton::Heathcote::13:13:24
Waverton::Gymea::11:12:13
Riverwood::Homebush::01:03:04
Oatley::Guildford::00:02:07
Ashfield::Padstow::00:07:54
Campsie::Strathfield::03:02:43
Campsie::Sutherland::00:07:54
Waverton::Gymea::13:13:24
Waverton::Heathcote::07:12:19
Waverton::Gymea::07:12:19
Riverwood::Homebush::00:07:54
Oatley::Guildford::00:07:54
Ashfield::Padstow::00:07:54
Campsie::Strathfield::12:12:12
Campsie::Sutherland::22:23:11
canterbury::Sydenham::00:00:01
Clyde::Turella::12:12:12
Werrington::Vineyard::00:07:54
Waverton::Vineyard::12:12:12
Campsie::Vineyard::22:23:11
Campsie::Waitara::07:12:19
Canterbury::Windsor::00:02:23                                                                                                                                                   
Clyde::Windsor::00:12:00
Waverton::Gymea::12:12:12
Waverton::Heathcote::11:01:21
Waverton::Gymea::12:32:32
Riverwood::Homebush::12:12:12
Oatley::Guildford::12:22:33
Ashfield::Padstow::00:00:01
Campsie::Strathfield::11:12:11
Campsie::Sutherland::00:00:01
Werrington::Vineyard::03:12:25
Waverton::Vineyard::00:07:54
Campsie::Vineyard::11:12:19
Campsie::Waitara::07:12:19
Canterbury::Windsor::03:12:25                                                                                                                                                   
Clyde::Windsor::11:12:11
Werrington::Westmead::00:00:01
Waverton::Windsor::00:00:01
Werrington::Vineyard::03:45:12
Waverton::Vineyard::00:00:01
Campsie::Vineyard::00:07:54
Campsie::Waitara::11:12:11
Canterbury::Windsor::03:45:41                                                                                                                                                   
Clyde::Windsor::03:45:03
Werrington::Westmead::00:00:01
Waverton::Windsor::03:12:25
Canterbury::Windsor::00:00:01                                                                                                                                                   
Clyde::Windsor::11:12:54
Werrington::Westmead::03:12:25
Waverton::Windsor::00:00:01
