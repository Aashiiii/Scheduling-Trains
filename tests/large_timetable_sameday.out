The next train to Vineyard from Campsie departs at 00:07:54
